ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/gui/JailPanel.java

```diff
@@ -16,24 +16,43 @@
     private static final long serialVersionUID = 9042633590279303353L;
     private SquareJLabel[][] mSquareLabels;
     private int mTeamIndex;
+    private int mJailDimension;
 
     public JailPanel(int totalPieces, int teamIndex) {
-        int jailDimension = (totalPieces / 4) + (totalPieces % 4);
+        mJailDimension = (totalPieces / 4) + (totalPieces % 4);
         setOpaque(false);
-        setLayout(new GridLayout(jailDimension, jailDimension));
-        setPreferredSize(new Dimension(jailDimension * 48, jailDimension * 48));
-        mSquareLabels = new SquareJLabel[jailDimension][jailDimension];
+        setLayout(new GridLayout(mJailDimension, mJailDimension));
+        setPreferredSize(new Dimension(mJailDimension * 48, mJailDimension * 48));
+        mSquareLabels = new SquareJLabel[mJailDimension][mJailDimension];
         mTeamIndex = teamIndex;
         setBorder(GuiUtility.createBorder(Messages.getString("PlayGamePanel.capturedPieces")));
-        setLayout(new GridLayout(jailDimension, jailDimension));
-        setPreferredSize(new Dimension((jailDimension + 1) * 25, (jailDimension + 1) * 25));
-        createGrid(jailDimension);
+        setLayout(new GridLayout(mJailDimension, mJailDimension));
+        setPreferredSize(new Dimension((mJailDimension + 1) * 25, (mJailDimension + 1) * 25));
+        createGrid(mJailDimension);
```

Comment:
if `mJailDimension` is a field now, do we still need to pass it in to `createGrid()` or can `createGrid()` just use it directly?